tsconfig.json

```diff
@@ -14,7 +14,6 @@
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
[Documentation]([documentation suggests](https://www.typescriptlang.org/tsconfig#suppressImplicitAnyIndexErrors)) - we were mass ignoring places where `any` was set by TS

Similar to this: https://github.com/TypeStrong/ts-loader/pull/1559